spec/features/events/event_rsvp_request_spec.rb

```diff
@@ -160,15 +163,15 @@
     describe "a new learn rsvp" do
       it "should show rails levels for rails events" do
         visit learn_new_event_rsvp_path(@event)
-        expect(page).to have_content Course.find_by_name('RAILS').levels[0][:title]
+        expect(page).to have_content @course.levels[0][:title]
       end
 
       it "should show frontend levels for frontend events" do
```

Comment:
probably don't need this to be a separate test if it's now just reading the value out of the DB